cypress/integration/e2eReactTest.js

```diff
@@ -0,0 +1,8 @@
+describe('test app', () => {
+  it('should write into search input', () => {
+    cy.visit('localhost:8080');
+    cy.get('#searchInput')
+      .type('iron man')
+      .should('have.value', 'iron man');
```

Comment:
Currently this test tests nothing except itself. It's better to check some functionality. For instance, you can check switching between pages.